packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -287,20 +287,30 @@ const ProductDetail = () => {
 
     /**************** Add To Cart ****************/
     const showToast = useToast()
-    const showError = () => {
+    const showError = (quantity) => {
         showToast({
-            title: formatMessage(API_ERROR_MESSAGE),
+            //title: formatMessage(API_ERROR_MESSAGE),
+            
+            // This error will inform the user that their specified input is no longer available
+            title: `The item is not available at the requested quantity of ${quantity}`,
```

Comment:
Thanks for contributing to the pwa-kit. A few points regarding this change:
- showError function is used at multiple places, so we choose a very generic message and try to be so specific.
- Any text in the pwa-kit is expected to wrapped in formattedMessage method for translations